import numpy.exceptions as ex

<<<<<<< HEAD
ex.AxisError(1.0)  # type: ignore[call-overload]
ex.AxisError(1, ndim=2.0)  # type: ignore[call-overload]
ex.AxisError(2, msg_prefix=404)  # type: ignore[call-overload]
=======
ex.AxisError(1.0)  # E: No overload variant
ex.AxisError(1, ndim=2.0)  # E: No overload variant
ex.AxisError(2, msg_prefix=404)  # E: No overload variant
>>>>>>> 9c5f2989031ba54019bec835b7ecb3f5768f2dcf
